concepts/team-changenotification-teams.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "Get change notifications for creation updation or deletion of teams using Microsoft Graph"
+description: "Get change notifications for creation updatation or deletion of teams using Microsoft Graph"
+author: "AbhishekAnand"
+localization_priority: Priority
+ms.prod: "microsoft-teams"
+ms.custom: scenarios:getting-started
+---
+
+# Get change notifications for teams using Microsoft Graph
+
+Change notifications enable you to subscribe to changes (create, update, and delete) to [team](/graph/api/resources/team?preserve-view=true). Change notifications provide a low latency model by allowing you to maintain a [subscription](/graph/api/resources/webhooks?preserve-view=true). You can also get the resource data in the notifications and therefore avoid calling the API to get the payload.
+
+>**Note:** The maximum time a subscription can last is 60 minutes; however, subscriptions can be renewed until the caller has permissions to access to resource.
+
+
+
+### Subscribe to changes in any team at tenant level
+
+To get change notifications for any change in teams' property across tenant, subscribe to `/teams`. This resource supports [including resource data](webhooks-with-resource-data.md) in the notification.
```

Comment:
'/teams' might give an impression that it covers everything under teams (be it channels, messages) , so we should clearly mention that this resource string is only for updates to properties of the team, not on a channel or message or member in the team